packages/mdc-ripple/README.md

```diff
@@ -150,6 +150,8 @@ Method Signature | Description
 `activate() => void` | Proxies to the foundation's `activate` method
 `deactivate() => void` | Proxies to the foundation's `deactivate` method
 `layout() => void` | Proxies to the foundation's `layout` method
+`handleFocus() => void` | Adds `BG_FOCUSED` class to the ripple surface
```

Comment:
I'd genericize these descriptions to simply state that they handle focus / blur events on the ripple surface (see also e.g. Text Field's descriptions of `handle` methods)